stan/math/prim/scal/prob/inv_chi_square_rng.hpp

```diff
@@ -2,51 +2,51 @@
 #define STAN_MATH_PRIM_SCAL_PROB_INV_CHI_SQUARE_RNG_HPP
 
 #include <stan/math/prim/scal/err/check_consistent_sizes.hpp>
-#include <stan/math/prim/scal/err/check_nonnegative.hpp>
-#include <stan/math/prim/scal/err/check_not_nan.hpp>
+#include <stan/math/prim/scal/err/check_finite.hpp>
 #include <stan/math/prim/scal/err/check_positive_finite.hpp>
-#include <stan/math/prim/scal/fun/constants.hpp>
-#include <stan/math/prim/scal/fun/multiply_log.hpp>
-#include <stan/math/prim/scal/fun/value_of.hpp>
-#include <stan/math/prim/scal/fun/gamma_q.hpp>
-#include <stan/math/prim/scal/fun/digamma.hpp>
 #include <stan/math/prim/scal/meta/length.hpp>
+#include <stan/math/prim/scal/meta/scalar_seq_view.hpp>
 #include <stan/math/prim/scal/meta/VectorBuilder.hpp>
-#include <stan/math/prim/scal/meta/return_type.hpp>
-#include <stan/math/prim/scal/fun/grad_reg_inc_gamma.hpp>
-#include <stan/math/prim/scal/meta/include_summand.hpp>
 #include <boost/random/chi_squared_distribution.hpp>
 #include <boost/random/variate_generator.hpp>
 
 namespace stan {
   namespace math {
-
     /**
-     * Return a pseudorandom Inverse chi squared variate with the nu degrees of
-     * freedom using the specified random number generator.
+     * Return a pseudorandom inverse chi squared variate with the nu
+     * degrees of freedom using the specified random number generator.
+     *
+     * nu can be a scalar, a std::vector, an Eigen::Vector, or an
```

Comment:
Same comment as before; somehting like "scalar or one-dimensional container" might be easiest for all of this going forward.